website/docs/zombie-flags.mdx

```diff
@@ -0,0 +1,107 @@
+- **Stale in all/any selected environments**: Choose whether to include flags that are stale in all or any of the selected environments.
+- **Filter by config or feature flag name**: Type at least the first letter of the feature flag or config name to auto-filter the results.
+
+<img alt="Navigating to Zombie Flags in the ConfigCat dashboard" src="/docs/assets/guide/my-zombie-flags-report/the-zombie-flags-page-in-the-dashboard_192dpi.png" width="1366" height="768"  decoding="async" loading="lazy" />
+
+By default, the Zombie Flags page will filter the zombie flags based on the filters set in the **My Zombie Flags Report** page.
+
+:::info
+If you have the [Code References](https://configcat.com/docs/advanced/code-references/overview/) feature enabled, you can also see where the flags are used in your code.
+:::
+
+### Using the Zombie Flags Report Email
+
+The Zombie Flags Report scans your ConfigCat dashboard and sends an email report of the feature flags that haven't been updated or toggled within a specified time frame.
+
+You could customize the report to include only the flags you want to track, similar to filtering in the dashboard.
+
+#### How to Set Up the Zombie Feature Flags Report
+
+1. Log in to the [ConfigCat dashboard](https://app.configcat.com/my-account/zombie-flags-report).
+2. Click the Organization name in the top-right corner.
+3. Select  **My Zombie Flags Report** from the dropdown menu. 
+
+<img alt="Navigating to My Zombie Flags Report" src="/docs/assets/guide/my-zombie-flags-report/navigating-to-my-zombie-flags-report_192dpi.png" width="1366" height="768"  decoding="async" loading="lazy" />
+
+#### The My Zombie Flags Report Page
+
+The email report could be customized as follows:
+
+- **Receive report via email:** Toggle this option to enable the email report, and specify how often you'd like to receive the report (e.g., daily, weekly, or monthly).
+
+- **Criteria:** Define how long a flag must be inactive before it's marked as stale.
+
+- **Scope:** Choose to include all feature flags or only the ones you're watching. 
+  - **Environment:** Select the ConfigCat [environment](https://configcat.com/docs/main-concepts/#environment) for which you want to analyze flags. 
+  - **Tags:** Select the flags you want to ignore in the report based on their tags.
+  - **Stale in all/any selected environments:** Choose whether to include flags that are stale in all or any of the selected environments.
+
+<img alt="My Zombie Flags Report page in the dashboard" src="/docs/assets/guide/my-zombie-flags-report/my-zombie-flags-report-page-in-the-dashboard_192dpi.png" width="1366" height="945" decoding="async" loading="lazy" />
+
+#### Sample Email Report
+
+Here's a preview of what the Zombie Flags report email looks like:
+
+<img alt="Zombie Flags Report Email" src="/docs/assets/guide/my-zombie-flags-report/zombie-flags-report-email_192dpi.png" width="1366" height="768" decoding="async" loading="lazy" />
```

Comment:
I think it would be better to blur out the testuserfactingbcc@configcat.com email address on the screenshot.